src/components/ArticlesCard.jsx

```diff
@@ -0,0 +1,15 @@
+export default function ArticlesCard({ArticlesName, ArticlesAuthor, ArticlesTopic, ArticlesCreatedAt, ArticlesVotes, ArticlesImg, ArticlesCommentCount}) {
+    return (
+        <div className="ArticlesCard">
+        <h3 className="ArticlesName">{ArticlesName}</h3>
+        <ul>
```

Comment:
When previewing this, you can still see the bullet points from the list. Look into changing the style for **list-style-type** to remove these (that's if you want to 😄)